processing-and-analysis/02-command-files/2s-practice-jackson/model-selection-NEW-dataset-jh.Rmd

```diff
@@ -0,0 +1,456 @@
+                           data = master)
+summary(urm_mod1.b)
+```
+
+#### AIC comparison of `urm_mod1.a` and `urm_mod1.b`
+
+```{r}
+AIC(urm_mod1.a, urm_mod1.b)
+```
+
+* The AIC *increases* by *more than 2* for `urm_mod1.b`, so we will retain `urm_id` in our model.
+
+* Now let's try removing `exp` from our model:
+
+```{r}
+urm_mod1.c <- lm(final_c ~ urm_id +
+                           satm_c + satv_c + aleksikc_c + hs_gpa_c,
+                           data = master)
+summary(urm_mod1.c)
+```
+
+#### AIC comparison of `urm_mod1.a` and `urm_mod1.c`
+
+```{r}
+AIC(urm_mod1.a, urm_mod1.c)
+```
+
+* The AIC *decreases* by *less than 2* for `urm_mod1.c`, which is the simplest model, so we will remove the `exp` parameter from our model.
+
+* Now let's try removing `satv_c`
+
+```{r}
+urm_mod1.d <- lm(final_c ~ urm_id +
+                           satm_c + aleksikc_c + hs_gpa_c,
+                           data = master)
+summary(urm_mod1.d)
+```
+
+#### AIC comparison of `urm_mod1.c` and `urm_mod1.d`
+
+```{r}
+AIC(urm_mod1.c, urm_mod1.d)
+```
+
+* The AIC *increases* by *more than 2* for `urm_mod1.d`, so we will retain the `satv_c` parameter in our model.
+* Our final model is `urm_mod1.c`, which is summarized above.
+
+## Model selection for `eop_id`
+
+### 1. Fixed-effects only model (EOP)
+
+```{r}
+eop_mod1 <- lm(final_c ~ exp * eop_id +
+                         exp + eop_id +
+                         satm_c + satv_c + aleksikc_c + hs_gpa_c,
+                         data = master)
+summary(eop_mod1)
+```
+
+### 2. Random intercepts model with `ta_sect`; REML = TRUE (EOP)
+
+```{r}
+eop_mod2 <- lmer(final_c ~ exp * eop_id +
+                           exp + eop_id +
+                           satm_c + satv_c + aleksikc_c + hs_gpa_c +
+                           (1 | ta_sect),
+                           data = master, REML = TRUE)
+summary(eop_mod2)
+```
+
+#### AIC comparison of `urm_mod1` and `urm_mod2`
+
+```{r}
+AIC(eop_mod1, eop_mod2)
+```
+
+* The AIC *increases* by *more than 2* for `eop_mod2`, so we will not retain random effects from `ta_sect` in our model.
+
+
+### 3. Fixed-effects only model with *no* demographic interactions (EOP)
+
+```{r}
+eop_mod1.a <- lm(final_c ~ exp + eop_id +
+                           satm_c + satv_c + aleksikc_c + hs_gpa_c,
+                           data = master)
+summary(eop_mod1.a)
+```
+
+#### AIC comparison of `eop_mod1` and `eop_mod1.a`
+
+```{r}
+AIC(eop_mod1, eop_mod1.a)
+```
+
+* The AIC *decreases* by *less than 2* for `eop_mod1.a`, which is the simplest model, so we will not include interactions from `eop_id` in our model.
+
+### 4. Removal of parameters one by one
+
+* Start by removing `exp` because it has the lowest t-value
+
+```{r}
+eop_mod1.b <- lmer(final_c ~ eop_id +
+                             satm_c + satv_c + aleksikc_c + hs_gpa_c +
+                             (1 | ta_sect),
+                             data = master, REML = TRUE)
```

Comment:
You still have a random intercept here, and you're using `lmer`. I think you copied the wrong model over.

Reply:
This also applies to `eop_mod1.c`.